client/src/components/BreadcrumbTrail.tsx

```diff
@@ -31,22 +34,35 @@ export const BreadcrumbTrail = ({
   ascendantObjects,
   parentField,
   isLink,
+  hideParents,
+  lastParentTask,
   style
 }: BreadcrumbTrailProps) => {
   const trail = utils.getAscendantObjectsAsList(
     leaf,
     ascendantObjects,
     parentField
   )
+  if (hideParents) {
+    // if hideParents is true, we remove all tasks up until the last parent task
+    const lastParentTaskIndex =
+      trail.findIndex(node => node.uuid === lastParentTask.uuid) || 0
```

Comment:
`lastParentTask` could be undefined here, for safety should use optional chaining.